python/TestHarness/TestHarness.py

```diff
@@ -508,8 +508,8 @@ def buildPBSBatch(self, output, tester):
     if 'command not found' in output:
       return ('QSUB NOT FOUND', '')
     else:
-      # Get the PBS Job ID using qstat
-      results = re.findall(r'JOB_NAME: (\w+) JOB_ID:.* (\d+).*TEST_NAME: (\S+)', output, re.DOTALL)
+      # Get the Job information from the ClusterLauncher. ClusterLuancher will return multiple lines if there are multiple tests (re.M)
+      results = re.findall(r'JOB_NAME: (\w+) JOB_ID:.* (\d+).*TEST_NAME: (\S+)', output, re.M)
```

Comment:
We'll you aren't using anchors ($, ^, etc) so the new flag isn't necessary. Removing DOTALL is probably correct though.